src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -113,11 +114,17 @@
     :fx [[:navigate-to-within-stack [:wallet-transaction-confirmation stack-id]]]}))
 
 (rf/reg-event-fx :wallet/get-suggested-routes
- (fn [{:keys [db now]} [amount]]
+ (fn [{:keys [db now]} [{:keys [amount]}]]
```

Comment:
this was just updated for readability
`[:wallet/get-suggested-routes 1]` vs `[:wallet/get-suggested-routes {:amount 1}]` in the second it's more apparent what 1 relates to.